app/javascript/widgets/landing_page/pages/home.jsx

```diff
@@ -95,12 +111,18 @@ const Home = () => {
         }}
       >
         <div className="tw-max-w-[1280px] tw-w-full tw-mx-auto tw-relative">
-          <WorldMapQASelect />
-          <WorldMap
-            className="tw-worldmap tw-w-[90%] tw-h-auto tw-mx-auto"
-            onClick={handleWorldMapClick}
-            data-selected-region={selectedRegion}
-          />
+          {hideDomainSwitcher === "false" || (hideDomainSwitcher === false && <WorldMapQASelect />)}
+          <div className="tw-relative tw-w-[90%] tw-h-auto tw-mx-auto">
+            {hideDomainSwitcher === "true" || hideDomainSwitcher === true ? (
```

Comment:
the only thing you're changing here is that the map doesn't react to clicks. So instead of this conditional render add the logic of not doing anything to the `handleWorldMapClick` event handler.

Reply:
ok thats also a good idea 👍